Chapters/1-1-Chapter1.md

```diff
@@ -0,0 +1,167 @@
+
+She moved to the mirror and I followed her. The growing light of Ara had been shining through the window, and it bounced off the mirror and into my eyes as I walked toward it. I put my arm up to shield the light while she stood just barely in it.
+
+She proceeded to wrap her cape around her shoulders and I took up the tying of her waistbelt behind her. The mauve color of Heiko shined proudly in the daylight. I remembered dying them in pohua, taking every care to ensure that the dye was consistent and fast to the cloth, but still I saw all of its imperfections whenever I looked at it. Melna didn't seem to mind.
+
+"This morning, you mentioned the cold air," I spoke hesitantly.
+
+She didn't respond.
+
+"The farmers have a responsibility, you know," I tried again to persuade her, "If a temperature drift is coming, we need to know so we aren't caught by an early-"
+
+"Liffa... we've been over this. If I had evidence of drift, I would tell you."
+
+She was a stone wall of secrets, and I was unsatisfied until I could break through. But it was unfair of me to keep pushing on it when I knew she was bound to her duties. I wasn't letting her go without a promise.
+
+I placed a hand on her bare shoulder, and pleaded, "Look me in the eye. Look at me and tell me you are not keeping secrets from your village. From your bounds."
+
+She looked at me through the mirror for a moment, searching for motives in an innocent face.
+
+"I don't have time for this," she said coldly.
+
+I let out a frustrated sigh and made my way to the doorway of her room.
+
+Before I could leave, she said, "Liffa..."
+
+I stopped without turning.
+
+"You know I love you."
+
+I left.
+
+---
+
+Kipnu was going over his schedule of patients in the same spot we had left him. He looked up and took a glance at me, burying his head in the schedule again.
+
+"Melna got you down?"
+
+"How could you tell?"
+
+"Ah you know."
+
+I waited for him to elaborate.
+
+"...Reasons."
+
+_Thanks Kip..._
+
+I mustered up a response, "Don't want to talk about it right now."
+
+He wasn't convinced, but he was more accepting about these things than I was.
+
+"Okay... suit yourself"
+
+I stooped at the table with him and a long awkward silence filled the room while I watched him write notes in his schedule. Melna came downstairs and looked us over. She had a confused look for a brief moment, but resumed her departure.
+
+"I'm headed to the train."
+
+"Have a good day," he chimed. I stayed silent.
+
+"I'll be back a bit late tonight. Don't wait up for me."
+
+Late wasn't particularly rare for her, but I wanted to pry.
+
+"Late? What for?" I asked.
+
+"Yansha has been pushing us on this grain resolution."
+
+I looked in her eyes more intently. Something about her eyes was very out of place. As if she had shut off her heart and it was wallowing in guilt. I stood up and approached her as she turned to leave the house.
```

Comment:
> As if she had shut off her heart and it was wallowing in guilt.

So Liffa is assuming Melna had done something wrong or withholding information she shouldn't?

Reply:
hmm yea this is some strange wording. I don't think Liffa would have any insight right now. Just would be suspicious.